docs/source/installation.rst

```diff
@@ -2,147 +2,123 @@ Installation
 ============
 
 Supported OS
-------------
+++++++++++++
 
-Currently, we only support ``MacOS`` and ``Linux`` operating systems. ``Windows``
-users have the possibility to install and use ``ivadomed`` via
-`Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/>`_. The steps below (about updating bashrc) are strongly recommended for MacOS users in the installation process but are optional for Linux users.
+Currently, ``ivadomed`` only supports GPU/CPU on ``Linux`` and ``Windows`` and CPU only on ``macOS`` and `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_.
 
-Open your bash/zsh script file with editor on your computer.
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using bash shell
-    ::
+You can setup ``ivadomed`` using either Conda or Venv:
 
-        vim ~/.bashrc
+.. tabs::
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+    .. tab:: Install via ``conda``
 
-Write in your .bashrc/.zshrc file with following line.
+        This is the easiest way for personal computers.
 
-::
+        1. Create new conda environment using ``environment.yml`` file
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+        ::
 
-Save this change and restart your terminal to apply the change.
+            conda env create --file environment.yml
 
-There are fundamentally two different approaches to install ``IvadoMed``:
+        2. Activate the new conda environment (default named ``ivadomed_env``)
 
-1) Install via Conda
-    This is the easiest way for personal computers.
+        ::
 
-2) Install via Venv/VirtualEnv
-    This is compatible with ComputeCanada cluster environment.
+            conda activate ivadomed_env
 
-Approach 1: Conda
-------------------
+    .. tab:: Install via ``venv``
 
-Step 1: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        1. Setup Python Venv Virtual Environment.
 
-::
+        ``ivadomed`` requires Python >= 3.6 and <3.9 (If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules (e.g. python 3.9) as `mentioned here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_ ). We recommend
+        working under a virtual environment, which could be set as follows:
```

Comment:
I think this "Compute Canada" bracketed text might be better off in the Compute Canada tab.

Reply:
Good point. They were originally sort of together. Now separated. Moving there.